pkg/config/config.go

```diff
@@ -27,7 +27,7 @@ type ConfigManager struct {
 type Config struct {
 	IgnoredVulns      []IgnoreEntry          `toml:"IgnoredVulns"`
 	PackageOverrides  []PackageOverrideEntry `toml:"PackageOverrides"`
-	LoadPath          string                 `toml:"LoadPath"`
+	LoadPath          string                 `toml:"-"`
```

Comment:
Could you add a comment for this?

Reply:
I've added a comment and moved the field to the end of struct to make it clearer that this is not a parsed field